src/VirtoCommerce.Platform.Core/Security/ApplicationUser.cs

```diff
@@ -25,6 +25,8 @@ public class ApplicationUser : IdentityUser, IEntity, IAuditable, ICloneable
         public virtual string CreatedBy { get; set; }
         public virtual string ModifiedBy { get; set; }
         public virtual IList<Role> Roles { get; set; }
+
+        [SwaggerIgnore]
```

Comment:
Do we need to set those attributes explicitly if we have ExcludeRedundantDepsFilter.cs which does the same?

Reply:
This filter removes a different swagger component. Maybe we can extend it.

Reply:
ExcludeRedundantDepsFilter removes model, instead of the member inside model.